platform/jvm/capture-plugin/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+ plugins {
+    alias(libs.plugins.kotlin)
+
+    id("dependency-license-config")
+    id("java-gradle-plugin")
+    id("maven-publish")
+    id("com.vanniktech.maven.publish") version "0.28.0" apply false
```

Comment:
you can just reference the libs catalog we use everywhere `alias(libs.plugins.maven.publish)`